### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by FranFin**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/174/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/193/)

**Review by pablogs9 — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/175/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/194/)

**Comment by pablogs9** [comment]
LGTM, warning is unrelated, if you are OK merge @jamoralp

File: README.md [hunk 1]
```diff
@@ -8,23 +8,34 @@
...
+This is made possible by the creation of *DDS Entities* on the *Agent* as a result of *Clients*' operations, such as *Participants*, *Topics*, *Publishers*, and *Subscribers*, which can interact with the DDS global dataspace.
+
+The communication between a *Micro XRCE-DDS Client* and a *Micro XRCE-DDS Agent* is achieved by means of several kinds of built-in transports: **UDPv4**, **UDPv6**, **TCPv4**, **TCPv6** and **Serial** communication. In addition, there is the possibility for the user to generate its own **Custom** transport.
+
+<p align="center"> <img src="docs/General.png" alt="Image" width="70%"/> </p>
+
+This repository contains the totality of the *eProsima Micro XRCE-DDS* products:
+
+- *Micro XRCE-DDS Client*
+- *Micro XRCE-DDS Client*
```
Comment (jamoralp) [inline README.md:33]:
```suggestion
- *Micro XRCE-DDS Agent*
```

File: README.md [hunk 2]
```diff
@@ -8,23 +8,34 @@
...
+- *Micro XRCE-DDS Gen*
```
Comment (jamoralp) [inline README.md:34]: I personally would expect this list ot be clickable. Therefore, write something as:
* [Micro XRCE-DDS Client](https://github.com/eProsima/Micro-XRCE-DDS-Client)

**Review by jamoralp — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/176/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/195/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/177/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/196/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/178/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/197/)

**Review by pablogs9 — APPROVED** [review]

**Comment by pablogs9** [comment]
If @jamoralp is ok with this we can merge.

**Review by jamoralp — APPROVED** [review]

**Pull request merged**